src/providers/settings.tsx

```diff
@@ -13,6 +13,11 @@ interface Settings {
    * Spoiler level for tags.
    */
   spoilerLevel: SpoilerLevel;
+
+  /**
+   * What images to show.
+   */
+  showNsfwImages: boolean;
```

Comment:
"What images to show" is strange comment for `boolean` variable, don't you think so 😄 ? Let's improve it

Reply:
Also rename this param to something like `isNsfwContentAllowed`